recipes/whatlies/meta.yaml

```diff
@@ -0,0 +1,278 @@
+{% set name = "whatlies" %}
+{% set version = "0.6.5" %}
+{% set build_num = 0 %}
+{% set name_ = name|lower %}
+
+######### SET DEPENDENCY VERSION REQUIREMENTS #########
+
+{% set pyver = ">=3.6,<3.8" %}
+
+# base_packages
+{% set libver_altair = ">=4.0.1" %} 
+{% set libver_bpemb = ">=0.3.0" %}
+{% set libver_gensim = "~=3.8.3" %}
+{% set libver_matplotlib_base = ">=3.2.0" %}
+{% set libver_scikit_learn = ">=0.24.0" %}
+
+# umap_packages
+{% set libver_umap_learn = ">=0.4.0" %}
+# rasa_packages
+{% set libver_rasa = ">=2.7.0" %}
+# fasttext_packages
+{% set libver_fasttext = "~=0.9.1" %}
+# spacy_packages
+{% set libver_spacy = ">=3.0.1" %}
+{% set libver_spacy_lookups_data = ">=0.3.2" %}
+# s2v_packages (+= spacy_packages)
+{% set libver_sense2vec = ">=1.0.2" %}
+# tf_packages
+{% set libver_tensorflow = ">=2.3.0,<2.5" %}
+{% set libver_tensorflow_text = ">=2.3.0" %}
+{% set libver_tensorflow_hub = ">=0.8.0" %}
+# transformers_dep
+{% set libver_transformers = ">=4.3.0" %}
+# sentence_tfm_dep
+{% set libver_sentence_transformers = ">=0.3.8" %}
```

Comment:
If you use a Jinja2 variable only once it's better to just keep the number and not add a variable. This only makes sense if you reuse them multiple times.

Reply:
Each of these have been used at least twice: once in `whatlies` subpackage (under `run_constrained`) and then again in another relevant subpackage. The idea is to only update the core/extra-dependency versions at the very top of the file **once only**, to facilitate ease of maintenance.